packages/tsconfig/base.json

```diff
@@ -16,7 +16,6 @@
     "strictNullChecks": true,
     "allowUnreachableCode": false,
     "allowUnusedLabels": false,
-    "exactOptionalPropertyTypes": true,
```

Comment:
Not trying to sneak this in without discussion 😆 I removed this to unblock my work, since it appears that [Zod doesn't support this option](https://github.com/colinhacks/zod/issues/1510) when using `.optional()`. The workaround for this would be ridiculously painful: I'd have to modify a schema way down into a deeply nested property, just to assert that a property is allowed to be present-but-undefined instead of not present. If I can't figure out a workaround by review time, let's discuss how to move forward.